### ISSUE TITLE
Narrator doesn't read the default way to open the submenu

### ISSUE THREAD

**Issue by yupasz** [microsoft/fluentui#14306]
### Environment Information

- **Package version(s)**: 7.123.9
- **Browser and OS versions**: Edge 84.0.522.44, Windows Server 2016 Datacenter 1607
### Please provide a reproduction of the issue in a codepen:
This issue is appears in the fluent UI documentation (Example of ContextualMenu with submenus):
[contextual menu](https://fabricweb.z5.web.core.windows.net/oufr/7.124.0/#/examples/contextualmenu)

1. Open the example in CodePen
2. Open Narrator
3. Navigate to the items in the drop down menu of 'Click for ContextualMenu' button
4. Observe the issue

#### Actual behavior:

When user navigates to sub menu of a contextual menu, the submenu can be selected when user press enter key or navigate with right/left arrow, but the narrator doesn't provide information about the same.

#### Expected behavior:
Narrator should narrates the default way to open the submenu or the sub menu should get opened as per narrator narration.

**Comment by paulgildea** [comment]
@yupasz Thanks for the feedback!

@smhigley FYI.

**Comment by khmakoto** [comment]
Hi @yupasz, if I'm understanding you correctly, your expectation is for us to include something announcing that the menu can be opened either via `Enter` or `Right Arrow` keys. If that's the case, I'm afraid we can't do that since that would be baking that information inside the component, creating localization issues. That should either be provided by the user (which we can change in the examples if you wanted us to) or be information deduced by the screenreader.

Right now the screenreader does tell you that there is a menu button there that can be expanded since it says that its current status is `collapsed`. Let us know if you want us to change the example to include this specific information, but otherwise, I don't think we're able to do anything more specific to the innards of the component.

**Comment by yupasz** [comment]
> Hi @yupasz, if I'm understanding you correctly, your expectation is for us to include something announcing that the menu can be opened either via `Enter` or `Right Arrow` keys. If that's the case, I'm afraid we can't do that since that would be baking that information inside the component, creating localization issues. That should either be provided by the user (which we can change in the examples if you wanted us to) or be information deduced by the screenreader.
> 
> Right now the screenreader does tell you that there is a menu button there that can be expanded since it says that its current status is `collapsed`. Let us know if you want us to change the example to include this specific information, but otherwise, I don't think we're able to do anything more specific to the innards of the component.

Yes, can you update the examples? Or if possible, can you also provide me a codepen link to check the behavior in advance?
Thanks!
